pkg/ast/ast.go

```diff
@@ -28,6 +28,16 @@ type Policy interface {
 	policyNode()
 }
 
+type Selector interface {
```

Comment:
please rename this to `WhereClause`

Reply:
ic, you can't name this `WhereClause` because that's already taken as a struct.

let's rename `Selector` to be `Conditions` - plural